Backend/src/main/java/Commands/UserCommands/GetUserData.java

```diff
@@ -1,28 +1,19 @@
-package Commands.UserCommands;
-
-import Entities.HttpResponseTypes;
-import Interface.ConcreteCommand;
-//import Models.User;
-import Models.Message;
-import com.arangodb.ArangoDB;
-import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import Database.ArangoInstance;
-import Models.UserData;
-public class GetUserData extends ConcreteCommand {
-
-    @Override
-    protected HttpResponseTypes doCommand() {
-        ArangoDB arangoDB=ArangoInstance.getArangoDB();
-        UserData userData = arangoDB.db(ArangoInstance.getDbName()).collection("users").getDocument(message.getUserID(), UserData.class);
-        JsonObject response = new JsonObject();
-        response.add("userData", jsonParser.parse(gson.toJson(userData)));
-        responseJson = jsonParser.parse(response.toString());
-        System.out.println(response);
-        return HttpResponseTypes._200;
-    }
-    @Override
-    public void setMessage(Message message) {
-
-    }
-}
+package Commands.UserCommands;
+
+import Interface.ConcreteCommand;
+//import Models.User;
+import Models.Message;
+import com.google.gson.Gson;
+import org.json.JSONObject;
+import Database.ArangoInstance;
+import Models.UserData;
+public class GetUserData extends ConcreteCommand {
+    @Override
+    public void setParameters(){
+        type="find";
+        model = "UserData";
+        inputParams= new String[]{"userData.id"};
```

Comment:
userData.id is the id of the user in the sql db, what you are looming for will be userData._key